sos/plugins/pulp.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2018 Red Hat, Inc., Jake Hunsaker <jhunsake@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+from pipes import quote
+
+
+class Pulp(Plugin, RedHatPlugin):
+    """Pulp platform"""
+
+    plugin_name = "pulp"
+    packages = ("pulp-server", "pulp-katello")
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/pulp/*.conf",
+            "/etc/pulp/server/plugins.conf.d/",
+            "/etc/default/pulp*",
+            "/var/log/httpd/pulp-http.log*",
+            "/var/log/httpd/pulp-https.log*",
+            "/var/log/httpd/pulp-http_access_ssl.log*",
+            "/var/log/httpd/pulp-https_access_ssl.log*",
+            "/var/log/httpd/pulp-http_error_ssl.log*",
+            "/var/log/httpd/pulp-https_error_ssl.log*"
+        ])
+
+        mtasks = self.build_mongo_cmd(
+            '\"DBQuery.shellBatchSize=${FOREMAN_DEBUG_MONGOTASKS:-200};; '
+            'db.task_status.find().sort({finish_time: -1})'
+            '.pretty().shellPrint()\"'
+        )
+
+        mres = self.build_mongo_cmd(
+            '\"db.reserved_resources.find().pretty().shellPrint()\"'
+        )
+
+        prun = self.build_mongo_cmd(
+            r'"db.task_status.find({state:{\$ne: \"finished\"}}).pretty()'
```

Comment:
Similarly here - please add `DBQuery.shellBatchSize=${FOREMAN_DEBUG_MONGOTASKS:-200};;` to see more pending `pulp` tasks.